### ISSUE TITLE
WebRTC undefined getters on IOS and Android 

### ISSUE THREAD

**Issue by brown62** [dint-dev/universal_html#29]
I'm implementing WebRTC and what I have so far builds and runs as expected when I debug with Chrome but when I try to run my code on an Android or IOS emulator I get a few dozen of the following errors and the build fails. 

```
lib/common/webRTC.dart:33:24: Error: Method not found: 'RtcPeerConnection'.
    myPeerConnection = RtcPeerConnection(rtcIceServers);

lib/screens/calling.dart:29:30: Error: The getter 'candidate' isn't defined for the class 'RtcPeerConnectionIceEvent'.
- 'RtcPeerConnectionIceEvent' is from 'package:universal_html/src/html_with_internals.dart' ('/usr/local/Caskroom/flutter/1.22.5/flutter/.pub-cache/hosted/pub.dartlang.org/universal_html-1.2.4/lib/src/html_with_internals.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'candidate'.

    'candidate': event.candidate.candidate,
                              
lib/screens/calling.dart:34:27: Error: The getter 'onTrack' isn't defined for the class 'RtcPeerConnection'.
- 'RtcPeerConnection' is from 'package:universal_html/src/html_with_internals.dart' ('/usr/local/Caskroom/flutter/1.22.5/flutter/.pub-cache/hosted/pub.dartlang.org/universal_html-1.2.4/lib/src/html_with_internals.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'onTrack'.
 
     peer.myPeerConnection.onTrack.listen((event) {
                                                    
```

I've tested a bunch of combinations but currently this is my environment:

```Flutter (Channel master, 1.26.0-18.0.pre.90, on macOS 11.1 20C69 darwin-x64, locale en-US)```
```Platform android-30, build-tools 30.0.3```
```Xcode 12.4, Build version 12D4e```
```import 'package:universal_html/html.dart';```
```
dependencies:
  flutter:
    sdk: flutter
  provider: ^4.3.2
  universal_html: ^1.2.4
```

and a snippet where I'm getting the errors:

```

peer.myPeerConnection.onIceCandidate.listen((event) {
      print('ice');
      if (event.candidate != null) {
        socket.send(jsonEncode({
          'type': 'candidate',
          'target': caller ?? contact,
          'candidate': event.candidate.candidate,
        }));
      }
    });

peer.myPeerConnection.onTrack.listen((event) {
      print('track');
    });
```

Any ideas on how to get it working on mobile?

Thanks!

**Comment by terrier989** [comment]
Thanks for the report! A pull request that adds the missing classes would be accepted.

**Comment by terrier989** [comment]
Fixed in v2.0.3.
